test/events.js

```diff
@@ -48,6 +51,7 @@ describe('the handleEvent() function', function () {
     });
     events.handleEvent({
       event: 'verified',
+      ts: Date.now() / 1000,
```

Comment:
As of train-98 all SQS events will have a `ts` field, so I updated this accordingly.